src/lib/View/SearchViewFilter.php

```diff
@@ -78,6 +78,7 @@ public function handleSearchForm(FilterViewBuilderParametersEvent $event): void
 
         $request = $event->getRequest();
 
+        /** @var array<string, mixed> $search */
```

Comment:
something strange happened here. This was resolved differently in `https://github.com/ibexa/search/pull/40/files#diff-8bad468f5d65de5f0490ca72ba5de084d0ddfa85f17b56fba1a73a01a64d5777R81` and I think that was better.